models/admin.js

```diff
@@ -32,6 +31,7 @@ const roleDelete = (role_id) => {
 };
 
 module.exports = {
+  fetchUser,
```

Comment:
Looks like this was prob the issue when not being able to pull users in admin settings.